Revit_Adapter/AdapterActions/Execute.cs

```diff
@@ -35,16 +36,7 @@ public partial class RevitAdapter
 
         public override Output<List<object>, bool> Execute(IExecuteCommand command, ActionConfig actionConfig = null)
         {
-            //Initialize Revit config
-            RevitPullConfig revitConfig = actionConfig as RevitPullConfig;
-
-            if (InternalAdapter != null)
-            {
-                InternalAdapter.RevitSettings = RevitSettings;
-                return InternalAdapter.Execute(command, revitConfig);
-            }
-
-            return null;
+            throw new NotImplementedException();
```

Comment:
Do you actually want an Exception to be thrown? Otherwise you can just delete the file `Execute.cs`.

By default, if invoked, it returns an empty output. This made me think that actually it should **also** do a `RecordError`. I will add that to the base Execute. I'd say, let's delete this file.

Reply:
Now targeted in https://github.com/BHoM/BHoM_Adapter/issues/207

Reply:
Done